core/api/views.py

```
@@ -1,17 +1,18 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
```

Ruff (0.8.2)

`django.shortcuts.render` imported but unused

Remove unused import: `django.shortcuts.render`

(F401)